students/johnpharmd/lesson08/circle_class.py

```diff
@@ -0,0 +1,54 @@
+import math
+
+
+class Circle:
+    """represents a simple circle, using properties, a class method, and
+    special methods"""
+
+    @classmethod
+    def sort(cls, radius_list):
+        return sorted(radius_list)
+
+    def __init__(self, radius):
+        self.radius = radius
+
+    def __str__(self):
+        return 'Circle with radius: ' + str(self.radius)
+
+    def __repr__(self):
+        return 'Circle(' + str(self.radius) + ')'
+
+    def __add__(self, other):
+        self.radius += other.radius
+        return self.__repr__()
+
+    def __mul__(self, value):
+        self.radius *= value
+        return self.__repr__()
+
+    def __rmul__(self, value):
+        return self.__mul__(value)
+
+    def __gt__(self, other):
+        return self.radius > other.radius
```

Comment:
If you use the `functools.total_ordering` decorator from the python standard library, you only need to define `__eq__` and `__lt__`, otherwise you should define all of the comparison operators